lib/rubocop/options.rb

```diff
@@ -512,7 +512,8 @@ module OptionsHelp
       version:                          'Display version.',
       verbose_version:                  'Display verbose version.',
       parallel:                         ['Use available CPUs to execute inspection in',
-                                         'parallel.'],
+                                         'parallel.  Or disable if set in the configuration.'],
```

Comment:
For consistency with the other toggleable switches:

```suggestion
                                         'parallel. Default is false.'],
```